CHANGELOG.md

```diff
@@ -36,9 +32,8 @@ bet we will fix some bugs and make a world even a better place.
   without decoding it if the line didn't end with a new line, a
   regression introduced in the 0.12.0 release.  ([#4361][])
 
-### Enhancements and new features
-
-?
+- The clone command failed to git-annex-init a fresh clone whenever
+  it considered to add the origin of the origin as a remote.  ([#4367][])
```

Comment:
CHANGELOG looks good to me aside form the missing link.  If you put in `(#4367)`, tools/link_issues_CHANGELOG will link-ify it for you.

Reply:
That's still confusing to me. All the other issues are referenced the exact same way.

Reply:
Ah, no. Now I found it (and got it hopefully). Thx!